.github/workflows/publish-charts.yml

```diff
@@ -32,6 +34,41 @@ jobs:
       - name: Install Helm
         uses: azure/setup-helm@v3
 
+      - name: Move ess-system out of charts dir
+        run: |
+          mv ./helm/ess-system ess-system-backup
+
+      - name: Run chart-releaser
+        uses: element-hq/chart-releaser-action@1389a6afc9af1b11ab9e79267f371d694e4995b2
+        env:
+          CR_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+        with:
+          charts_dir: ./helm
+          skip_existing: true
+          skip_cr_install: true
+
+      - name: Move ess-system back to charts dir
+        run: |
+          mv ess-system-backup ./helm/ess-system
+          rm -rf ./helm/operator
+          rm -rf ./helm/updater
+
+      - name: Wait for gh-pages to be up to date
+        run: |
+          URL="https://element-hq.github.io/ess-starter-edition-core/element-operator-${VERSION}.tar.gz"
+          STATUS=0
+
+          while [ $STATUS -ne 200 ]; do
+              echo "Checking URL: $URL"
+              STATUS=$(curl -s -o /dev/null -w "%{http_code}" $URL)
+              if [ $STATUS -ne 200 ]; then
+                  echo "Status: $STATUS. Not 200 OK, retrying in 5 seconds..."
+                  sleep 5
+              else
+                  echo "Status: $STATUS. URL is accessible."
+              fi
+          done
+
       - name: Run chart-releaser
```

Comment:
Will this release the operator and updater charts a 2nd time?

Reply:
No because they are removed from the directory when ess-system is moved back here